### ISSUE TITLE
Delete/Reset quota for group

### ISSUE THREAD

**Issue by Patrock** [nextcloud/groupquota#18]
hello, how to reset or delete a quota to its defaults ?

thx 

Patrock

**Comment by WNYmathGuy** [comment]
Adding onto this and requesting a feature, I got myself in trouble using it.
I spun up a NC 22 recently. First time ever using this app. Had about 100 groups to put a default quota on. The group names had whitespaces in their names. Turned out to be a bad idea. I deleted the groups and recreated them with underscores where there were spaces. I forgot about making all the default quota's and now they are still listed but I can't remove them or set them to 0GB.

I looked around with `grep` and so forth to try to find a way to remove the now erroneous entries, then gave up and came here.

Can you have a cleanup operation that looks for detached entries that trigger's off the normal chron job? And maybe for the guy above have it where `php occ group_default_quota:set "Unit 002" 0GB` or `php occ group_default_quota:clear "Unit 002"` would remove a quota?

**Comment by DerVO** [comment]
I'm using a hosted Nextcloud instance, and I am not able to use the CLI client.

I looked to an API response for a group w/o quota:

`$ curl -u admin:pass -H 'OCS-APIRequest: true' https://my.instance.com/apps/groupquota/quota/group-name`

```xml
<?xml version="1.0"?>
<ocs>
 <meta>
  <status>ok</status>
  <statuscode>100</statuscode>
  <message>OK</message>
  <totalitems></totalitems>
  <itemsperpage></itemsperpage>
 </meta>
 <data>
  <quota_bytes>-3</quota_bytes>
  <quota_human>?</quota_human>
  <used_bytes>16322021517</used_bytes>
  <used_human>15.2 GB</used_human>
  <used_relative>-544067383900</used_relative>
 </data>
</ocs>
```

So I have tried to set the quota to `-3` for my group, and it worked: the group quota was no longer in effect.

`$ curl -u admin:pass -H 'OCS-APIRequest: true' https://my.instance.com/apps/groupquota/quota/group-name  -X POST -d 'quota=-3'`

**Comment by RegExe** [comment]
This needs to be a really high priority as quotas utterly bug out when a user is in multiple groups.  Especially now that there are group shared folders and you must be in multiple groups to access multiple group folders - the lack of a feature to unbind the group quotas from a group is a showstopper.

Also, just an idea, why not;

`$user->SetQuota() = reset ( rsort ( $groupquotas[$userid] ) ); // this gets the greatest quota...
`

**Comment by joshtrichards** [comment]
Fixed in #35
